Library/ViewModels/RewardAddOnSelectionViewModelTests.swift

```diff
@@ -1107,25 +1107,27 @@ final class RewardAddOnSelectionViewModelTests: TestCase {
     self.vm.inputs.configure(with: data)
     self.vm.inputs.viewDidLoad()
 
-    XCTAssertEqual(["Add-Ons Page Viewed"], self.dataLakeTrackingClient.events)
```

Comment:
I remember we spoke about this last week. This should not have been deleted and moved down. Please look into this as I'm fairly certain it is the designed functionality. It is probably directly related to the changes applied to the `RewardAddOnSelectionViewModel`.

Reply:
We determined that because the shipping rule is required before the event is fired, it makes sense that this assertion is moved lower.